beamer/chain.py

```diff
@@ -281,6 +281,11 @@ def process_claims(context: Context) -> None:
 
 def fill_request(request: Request, context: Context) -> None:
     block = context.latest_blocks[request.target_chain_id]
+    time_until_expiry = request.valid_until - block["timestamp"]
```

Comment:
I think `time.time()` instead of `block["timestamp"] would be more appropriate and consistent here. 
the timestamp of the latest block is likely to be in the past. This means that the actual expiry time is less than the calculated one.